src/asm/lexer.c

```diff
@@ -202,6 +202,7 @@ static struct KeywordMapping {
 	{"STRCAT", T_OP_STRCAT},
 	{"STRUPR", T_OP_STRUPR},
 	{"STRLWR", T_OP_STRLWR},
+	{"STRREP", T_OP_STRREP},
```

Comment:
As I said elsewhere https://github.com/gbdev/rgbds/issues/660#issuecomment-752250521, I don't like this name for being ambiguous (`REP` could imply `REPEAT`, especially with `REPT` already being a thing). My suggestion is `REPLACE`.

Reply:
Just "`REPLACE`", or "`STRREPLACE`" to go with the rest?

Reply:
`REPLACE` is shorter, and I think we don't need to repeat that it's a string function

Reply:
It's just that `REPLACE` is a pretty generic term, more likely to be used as a label or constant name. And the other functions aren't named `LENGTH`, `UPPER`, `LOWER`, etc. (Nor `SUBSTR`/`SUBSTRING` instead of `STRSUB`, which is actually the one I sometimes think is "substitute" instead of "substring", like Ruby `gsub`.)

Reply:
`SUBST`, then? Or `STRSUBST`, if you don't think the other one fits.

Reply:
I didn't mean it's worth renaming `STRSUB`, just that it's an ambiguous function name that already exists. And having `STRSUB` coexist with `STRSUBST` would be if anything more confusing.

Reply:
I didn't mean to rename `STRSUB`—it's used enough that I want to avoid renaming it if I can. Then, just `SUBST`?

Reply:
That's still having two string-related functions that mean different things by "SUB". For what it's worth, the first Google result for "strrep" is the Matlab function that's just like this one: "`newStr = strrep( str , old , new )` replaces all occurrences of `old` in `str` with `new` ."

Reply:
I'll call for external opinion there, then. Paging @AntonioND @aaaaaa123456789 and @mid-kid.

Reply:
Hmm, hgbasm calls this `STRRPL`.

Reply:
Fine by me.

Reply:
Done; it's `STRRPL`.